eslzArm/managementGroupTemplates/policyDefinitions/china/mcPolicies.json

```diff
@@ -6920,7 +6920,7 @@
         },
         {
           "properties": {
-            "description": "This policy denies the creation of a subsnet with out an Network Security Group. NSG help to protect traffic across subnet-level.",
+            "description": "This policy denies the creation of a subnet with out an Network Security Group. NSG help to protect traffic across subnet-level.",
```

Comment:
"This policy denies the creation of a subnet without a Network Security Group (NSG). NSG helps to protect traffic across subnet-level."

Reply:
This is already as per your suggestion in the latest commit in this branch: https://github.com/withstu/Enterprise-Scale/blob/da051b551a26e28a2ee248e64a8e73e4a8e154f0/eslzArm/managementGroupTemplates/policyDefinitions/china/mcPolicies.json#L6923